<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="lwo1-lhp-f71232" name="com.vidyo.service.statusnotify.StatusNotifyListenerTests" tests="3" failures="0" timestamp="17 May 2018 13:53:28 GMT" time="0.685" errors="0">
  <testcase name="testCallExternalRestServiceWithNoRetry" time="0.146" classname="com.vidyo.service.statusnotify.StatusNotifyListenerTests"/>
  <testcase name="testCallExternalRestServiceThrowingException" time="0.206" classname="com.vidyo.service.statusnotify.StatusNotifyListenerTests"/>
  <testcase name="testCallExternalRestServiceSuccess" time="0.333" classname="com.vidyo.service.statusnotify.StatusNotifyListenerTests"/>
</testsuite> <!-- com.vidyo.service.statusnotify.StatusNotifyListenerTests -->
